backend/danswer/chat/chat_utils.py

```diff
@@ -71,7 +74,7 @@ def build_context_str(
 def build_chat_system_message(
     prompt: Prompt,
     context_exists: bool,
-    llm_tokenizer: Callable,
+    tokenizer_encode_func: Callable,
```

Comment:
A bit confusing without llm in the name as we use other tokenizers quite a bit as well. Can we call it llm_tokn_enc_func or something?

Reply:
can we check if litellm can be used to get the right tokenizer

Reply:
They do have this - will add in a follow up